provider/cmd/pulumi-resource-cloudflare/bridge-metadata.json

```diff
@@ -1956,10 +2048,12 @@
                 }
             },
             "cloudflare_zone_dnssec": {
-                "current": "cloudflare:index/zoneDnssec:ZoneDnssec"
+                "current": "cloudflare:index/zoneDnssec:ZoneDnssec",
+                "majorVersion": 5
             },
             "cloudflare_zone_lockdown": {
                 "current": "cloudflare:index/zoneLockdown:ZoneLockdown",
+                "majorVersion": 5,
```

Comment:
Hmm so should these majorVersion: 5 changes be pushed back onto the branch to inform auto-aliasing major version heuristic better?